Dockerfile

```diff
@@ -62,6 +61,14 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install lxc
+RUN mkdir -p /usr/src/lxc
+RUN curl -sSL https://linuxcontainers.org/downloads/lxc/lxc-1.0.7.tar.gz | tar -v -C /usr/src/lxc/ -xz --strip-components=1
+RUN cd /usr/src/lxc \
+	&& ./configure --prefix=/usr/ \
```

Comment:
well their default for .configure is /usr/local as the prefix and I couldn't get it to work with that.... idk